src/main/java/io/epirus/console/account/AccountManager.java

```diff
@@ -53,59 +54,123 @@ public static void main(final CliConfig config, final String[] args) throws IOEx
                     if (tokenError == null || tokenError.isEmpty()) {
-                        System.out.println("Could not retrieve token. Try again later.");
+                        Console.exitError("Could not retrieve token. Try again later.");
                     } else {
-                        System.out.println(tokenError);
+
+                        Console.exitError(tokenError);
                     }
                     return;
                 }
                 String token = responseJsonObj.get("token").getAsString();
                 config.setLoginToken(token);
+                config.save();
                 System.out.println(
                         "Account created successfully. You can now use Epirus Cloud. Please confirm your e-mail within 24 hours to continue using all features without interruption.");
             } else {
-                System.out.println("Account creation failed. Please try again later.");
+                Console.exitError("Account creation failed. Please try again later.");
             }
 
         } catch (IOException e) {
-            System.out.println("Could not connect to the server.\nReason:" + e.getMessage());
+            Console.exitError("Could not connect to the server.\nReason:" + e.getMessage());
         }
     }
 
-    protected final Response executeClientCall(Request newAccountRequest) throws IOException {
+    private final Response executeClientCall(Request newAccountRequest) throws IOException {
         return client.newCall(newAccountRequest).execute();
     }
 
-    protected final RequestBody createRequestBody(String email) {
+    final RequestBody createRequestBody(String email) {
 
         return new FormBody.Builder().add("email", email).build();
     }
 
-    protected final Request createRequest(RequestBody accountBody) {
+    final Request createRequest(RequestBody accountBody) {
 
         return new Request.Builder()
-                .url(String.format("%s/auth/realms/EpirusPortal/web3j-token/create", CLOUD_URL))
+                .url(String.format("%s/auth/realms/EpirusPortal/web3j-token/create", cloudURL))
                 .post(accountBody)
                 .build();
     }
 
+    public void checkIfAccountIsConfirmed() throws IOException, InterruptedException {
+        Request request =
+                new Request.Builder()
+                        .url(
+                                cloudURL
+                                        + "/auth/realms/EpirusPortal/web3j-token/status/"
+                                        + config.getLoginToken())
+                        .get()
+                        .build();
+        System.out.println("Checking if the account is activated...");
+        int tries = 5;
+        while (tries-- > 0) {
+            if (userConfirmedAccount(request)) {
+                System.out.println("Account is active.");
+                return;
+            } else {
+                Thread.sleep(5000);
```

Comment:
i'm a bit confused by this. If the account is active it won't sleep. It will only sleep if the account is not active. On the first point 25 seconds is a bit low i will increase the number of tries here in order to allow the user more time to activate the account.